x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -23,6 +23,7 @@ export interface EndpointAction {
   input_type: 'endpoint';
   agents: string[];
   user_id: string;
+  timeout?: number;
```

Comment:
can you add a comment above this new attribute? When I first looked at it, it thought it would be milliseconds, but I see from the code below it seems to be seconds.
Maybe:

```suggestion
  /** The number of seconds that the Elastic Agent (on the host) should wait to send back an action result before it timesout */
  timeout?: number;
```

Reply:
Will this new attribute also be present in the Action Result? if so, we should add it to that type as well.

Reply:
👍 added a comment on the attribute and looks like it's [not included in the action result](https://github.com/elastic/fleet-server/blob/a9be0dbd2717d3359e5fa4aff135dc6effb8fb46/model/schema.json#L67). thanks for pointing that out for me to check.